code/game/objects/items/storage/dakis.dm

```diff
@@ -0,0 +1,46 @@
+//////////////////////////////////
+ //dakimakuras
+ //////////////////////////////////
+
+/obj/item/storage/daki
+	name = "dakimakura"
+	desc = "A large pillow depicting a girl in a compromising position. Featuring as many dimensions as you."
+	icon = 'icons/obj/daki.dmi'
+	icon_state = "daki_base"
+	lefthand_file = "icons/mob/inhands/misc/daki_lefthand.dmi"
+	righthand_file = "icons/mob/inhands/misc/daki_righthand.dmi"
+	slot_flags = SLOT_BACK
+	storage_slots = 3
+	w_class = WEIGHT_CLASS_NORMAL
+	var/cooldowntime = 20
+	var/static/list/dakimakura_options = list("Callie","Casca","Chaika","Elisabeth","Foxy Grandpa","Haruko","Holo","Ian","Jolyne","Kurisu","Marie","Mugi","Nar'Sie","Patchouli","Plutia","Rei","Reisen","Naga","Squid","Squigly","Tomoko","Toriel","Umaru","Yaranaika","Yoko") //Kurisu is the ideal girl." - Me, Logos. //how could someone make something so good then say some retarded shit like this
+
+/obj/item/storage/daki/attack_self(mob/living/user)
+	var/body_choice
+	var/custom_name
+
+	if(icon_state == "daki_base")
+		body_choice = input("Pick a body.") in dakimakura_options
+		icon_state = "daki_[body_choice]"
```

Comment:
needs sanity checks after input

Reply:
?

Reply:
a sanity check is checking if something is still viable, it prevents runtimes.